mlflow/utils/uri.py

```diff
@@ -126,6 +127,12 @@ def is_databricks_unity_catalog_uri(uri):
     return scheme == _DATABRICKS_UNITY_CATALOG_SCHEME or uri == _DATABRICKS_UNITY_CATALOG_SCHEME
 
 
+# Update scheme parsing if changes are made to the OSS URI (this is an initial proof of concept)
+def is_oss_unity_catalog_uri(uri):
+    scheme = urllib.parse.urlparse(uri).scheme
+    return scheme == "uc" or uri == "uc"
```

Comment:
The naming of this function is throwing me. If it were just returning `return scheme == "uri"`, then this makes sense. But we're allowing an alias string to stand-in for a uri that we're constructing if this is running in Databricks. 
Would it make things a bit more clear to have separate handling for the alias resolution?

Reply:
I think this function was intended to just check the scheme. I've mistakenly included the uri == "uc" and will remove it. We are using uc:databricks-uc for the alias string to stand-in when running on databricks, so just checking if the scheme == "uc" is good enough. Let me know if I'm misunderstanding you.

Reply:
Nope, you got it! :)